myauth/ldap.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+from django.utils import timezone
+from django.conf import settings
+from django.core.exceptions import PermissionDenied as DjangoPermissionDenied
+from django_etebase.utils import CallbackContext
```

Comment:
These (`django_etebase`, `myauth`, `etebase_fastapi`) all need to be updated according to #140.